oddbird/Metecho
# [clean-up-devhub] Clean up user interactions with scratch orgs.

## Cute animal pic
_Because everyone likes pictures of animals._

![clownfish](http://www.animalspot.net/wp-content/uploads/2012/05/Anemonefish-Photos.jpg)

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/dZtQ9YH9

## Description
_The commit messages say what you did; this should explain why and how._

- Require SF auth to re-check scratch org for unsaved changes.
- Do not require Dev Hub for capturing changes from a scratch org.

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally._

1. Create a scratch org with unsaved changes.
2. Log in with a SF org that does not have Dev Hub enabled.
3. Capture those changes. [It now works.]
4. Disconnect from SF and attempt to check for unsaved changes. [It now asks for SF connection first.]

---

## Reviewer tasks:
